include/domains/wallinDomain.hpp

```diff
@@ -85,14 +80,14 @@ namespace ghost
     inline pair<int, int> getStartingTile()	const { return startingTile; }
     inline pair<int, int> getTargetTile()	const { return targetTile; }
            
-    inline int		 getNberRows()	const { return nRow_; }
-    inline int		 getNberCols()	const { return mCol_; }
+	inline int		 getNberRows()	const { return maxY_; }
+	inline int		 getNberCols()	const { return maxY_; }
     inline bool		 hasFailure()	const { return !failures_.empty(); }
     inline mapFail	 failures()	const { return failures_; }
 
-    inline pair<int, int> lin2mat( int p )	      const {return make_pair(p / mCol_, p % mCol_);}
-    inline int		  mat2lin( int row, int col ) const {return row * mCol_ + col;}
-    inline int		  mat2lin( pair<int, int> p ) const {return p.first * mCol_ + p.second;}
+	inline pair<int, int> lin2mat(int p)	      const { return make_pair(p % maxY_, p / maxY_); }
```

Comment:
Yes, I meant the space after "(" and before ")". That's my personal spacing policy. :p
I am pretty sure VS is configurable to handle that (at least, MonoDevelop for C# is), but since my code is not under a VS project, you have either to make that into a project or to configure your VS to apply this policy for all C++ code, which is painful in your side. I guess re-modifying by hand modifications from VS would be faster and easier (but still a bit tedious, sorry about that).